src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAGenerateAppConfig.cs

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using NuGet.Frameworks;
+using System.Linq;
+using System.Xml.Linq;
+using Xunit;
+
+namespace Microsoft.NET.Build.Tasks.UnitTests
+{
+    public class GivenAGenerateAppConfig
```

Comment:
Rename this class/file to `GivenAGeneratedAppConfig` (it's missing a "d").

Reply:
Ah, forget to run through a round of spell checker

Reply:
fixed